Editor/SceneViewToolbar/SceneViewToolbar.cs

```diff
@@ -136,6 +137,19 @@ private static void OnSceneGUI(SceneView sceneView)
                         }
                     }
 
+                    if (Preferences.showPipelineSwitcher)
```

Comment:
This code is probably a leftover which is not related to your PR. Please remove.

> 
> _Notes :_
> 
> If you need to implement it in your project I suggest you use the `OnSceneViewToolbarGUI`
>  
> If you want to include a render pipeline switcher, I'd advise you don't use the current GraphicsSettings.renderPipelineAsset but instead use quality settings.
> 
> Also this code only takes URP and BiRP into account, so it is pretty specific.